tests/evaluator/msbg/test_smearing.py

```diff
@@ -2,15 +2,41 @@
 import numpy as np
 import pytest
 
-from clarity.evaluator.msbg.smearing import Smearer, make_smear_mat3
+from clarity.evaluator.msbg.smearing import Smearer, audfilt, make_smear_mat3, smear3
 
-# audfilt
 
-# make_smear_mat3
+def test_audfilt():
+    """Test the auditory filter function"""
+    sample_freq = 44100
+    r_lower = 0.5
+    r_upper = 1.5
+    n_taps = 128
+    filter_params = audfilt(rl=r_lower, ru=r_upper, sampfreq=sample_freq, asize=n_taps)
+    assert filter_params.shape == (n_taps, n_taps)
+    assert np.sum(np.abs(filter_params)) == pytest.approx(19.879915844855944)
 
-# smear3
 
-# Smearer
+def test_make_smear_mat3_valid_input():
+    """Tests that make_smear_mat3 returns matrix with the correct dimensions"""
+    r_lower = 0.5
+    r_upper = 1.5
+    sample_freq = 44100
+    f_smear = make_smear_mat3(rl=r_lower, ru=r_upper, fs=sample_freq)
+    assert f_smear.shape == (256, 256)
+    assert np.sum(np.abs(f_smear)) == pytest.approx(2273.976168294156)
+
+
+def test_smear3():
+    """Test smear3"""
+    np.random.seed(0)
+    r_lower = 0.5
+    r_upper = 1.5
+    sample_freq = 44100
```

Comment:
Minor point as above, could set `r_lower`, `r_upper` and `sample_freq` outside of all three functions.